ci-default.yml

```diff
@@ -0,0 +1,118 @@
+deps:
+# COMMON
+- name: pmm
+  branch: PMM-2.0
+  path: sources/pmm/src/github.com/percona/pmm
+  url: https://github.com/percona/pmm
+  component: common
+
+# CLIENT
+- name: pmm-admin
+  branch: master
+  path: sources/pmm-admin/src/github.com/percona/pmm-admin
+  url: https://github.com/percona/pmm-admin.git
+  component: client
+- name: pmm-agent
```

Comment:
Can we put a space line between two neighboring `-name` sections for better readability?

Reply:
+1